its/projects/NuGet.config

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+      <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```

Comment:
Looks good to unblock master.
Will you follow-up to add the repox credentials? 

basically replace this with 
```
  <packageSources>
    <clear />
    <add key="Repox" value="https://repox.jfrog.io/artifactory/api/nuget/v3/nuget/index.json" protocolVersion="3" />
  </packageSources>
  <packageSourceCredentials>
    <Repox>
      <add key="Username" value="%ARTIFACTORY_USER%" />
      <add key="ClearTextPassword" value="%ARTIFACTORY_PASSWORD%" />
    </Repox>
  </packageSourceCredentials>
```

Reply:
I will do this in the current PR. My previous understanding was that for ITs we don't have to do this.